unasyncd/transformers.py

```diff
@@ -208,6 +208,12 @@ def _run_ruff(self, source: str, output: str, tree: cst.Module) -> str:
                 raise ChildProcessError(f"Error calling ruff: {stderr}")
             return stdout
 
+    def _ruff_fix(self, source: str, output: str, tree: cst.Module) -> str:
+        return self._run_ruff(source, output, tree, mode="fix")
+
+    def _ruff_format(self, source: str, output: str, tree: cst.Module) -> str:
+        return self._run_ruff(source, output, tree, mode="format")
+
```

Comment:
```suggestion
    def _ruff_fix(self, source: str, output: str, tree: cst.Module) -> str:
        return self._run_ruff(source, output, tree, cmd=["check", "--fix", "--fix-only"])

    def _ruff_format(self, source: str, output: str, tree: cst.Module) -> str:
        return self._run_ruff(source, output, tree, cmd=["format"])
```
together with the changes proposed above